src/packages/piral-oidc/src/setup.ts

```diff
@@ -15,41 +22,57 @@ export function setupOidcClient(config: OidcConfig): OidcClient {
     responseType,
     scopes,
     restrict = false,
+    appUri,
+    logLevel,
   } = config;
+
   const userManager = new UserManager({
     authority: identityProviderUri,
     redirect_uri: redirectUri,
     silent_redirect_uri: redirectUri,
+    popup_redirect_uri: redirectUri,
     post_logout_redirect_uri: postLogoutRedirectUri,
     client_id: clientId,
     client_secret: clientSecret,
     response_type: responseType,
     scope: scopes?.join(' '),
   });
 
-  if (process.env.NODE_ENV === 'development') {
+  if (logLevel !== undefined) {
+    Log.logger = console;
+    Log.level = logLevel;
+  } else if (process.env.NODE_ENV === 'development') {
     Log.logger = console;
     Log.level = Log.DEBUG;
   }
 
-  userManager.signinRedirectCallback();
-  userManager.signinSilentCallback();
-  userManager.signoutRedirectCallback();
+  if (window.location.pathname === new URL(userManager.settings.post_logout_redirect_uri).pathname) {
```

Comment:
This pattern shows up quite a lot.

A helper could be cool here:

```ts
function checkLocationPath(targetUri: string) {
  return window.location.pathname === new URL(targetUri).pathname;
}
```

Usage:

```ts
if (checkLocationPath(userManager.settings.post_logout_redirect_uri) {
 // ...
}
```

Reply:
Created `doesWindowLocationMatch(path)`